docker-compose/versions/camunda-8.8/.console/application.yaml

```diff
@@ -0,0 +1,73 @@
+camunda:
+  console:
+    customerId: "camunda-platform"
+    installationId: "camunda-platform"
+    telemetry: "disabled"
+    managed:
+      method: "plain"
+      releases:
+        - name: "camunda-platform"
+          namespace: "camunda-platform"
+          version: "8.8.0-alpha7"
+          tags:
+            - "dev"
+            - "local"
+          components:
+            - name: "Console"
+              id: "console"
+              version: "8.8.0-alpha7"
+              url: "http://localhost:8089"
+              readiness: "http://console:8080/health/readiness"
+              metrics: "http://console:9100/prometheus"
+            - name: "Zeebe Gateway"
+              id: "zeebeGateway"
+              version: "8.8.0-alpha7"
```

Comment:
Regarding maintenance. Do we need to update these versions on each new release now?